vega_sim/scenario/scenario.py

```diff
@@ -97,6 +101,10 @@ def run_iteration(
     def get_snitch(self) -> Optional[Snitch]:
         return self.agents.get("snitch")
 
+    def get_agents(self) -> dict:
+        snitch = self.get_snitch()
+        return snitch.agents if snitch is not None else {}
```

Comment:
This seems slightly confusing that it defaults to an empty dictionary? Why not `self.agents` (and what is the reason to use this rather than just `self.agents`? I can't see it used in the PR

Reply:
Ah thanks this is left over from an earlier approach!

 The scenario `self.agents` dict contains the `Snitch` whilst the snitches `self.agents` dict does not. In the earlier approach having that Snitch was causing a problem but this whole method is now redundant.

Will removed it 👍